src/ce.js

```diff
@@ -132,47 +133,45 @@ var ce = {
         return value;
     },
 
-    _sanitizeInputValue: function(input, value) {
+    _includeProperty: function(input, value) {
         var classes = (input.className || '').split(' ');
-        if (_.includes(classes, 'mp-never-strip-value')) { // never sanitize inputs with class "mp-never-strip-value"
-            return value;
-        } else if (_.includes(classes, 'mp-always-strip-value')) { // always sanitize fields with class "mp-always-strip-value"
-            return '[stripped]';
+        if (_.includes(classes, 'mp-always-include-value')) { // never sanitize inputs with class "mp-never-strip-value"
```

Comment:
Talked to @jbwyme about this -- they never were documented, and Autotrack is still beta, and we're not aware of anyone using them. so we think they're ok to remove.